assignment-1/Functions.java

```diff
@@ -0,0 +1,271 @@
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toMap;
+
+public class Functions {
```

Comment:
Should have used more meaningful class name, This does not give any information about the functionality of the class.